Sources/CGFloatExtensions.swift

```diff
@@ -13,7 +13,7 @@ extension CGFloat {
     public var center: CGFloat { return (self / 2) }
 }
 
-/// EZSwiftExtensions
+/// EZSE: Converts angle degrees to radians
```

Comment:
@thellimist I think "angle degree's" would be better because anything can be a degree, like temperature